static/app/components/onboarding/gettingStartedDoc/useSourcePackageRegistries.tsx

```diff
@@ -0,0 +1,36 @@
+import {Client} from 'sentry/api';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+type ReleaseRegistrySdk = Record<
+  string,
+  {
+    canonical: string;
+    main_docs_url: string;
+    name: string;
+    package_url: string;
+    repo_url: string;
+    version: string;
+  }
+>;
+
+// This exists because /extensions/type/search API is not prefixed with
+// /api/0/, but the default API client on the abstract issue form is...
+const API_CLIENT = new Client({baseUrl: '', headers: {}, credentials: 'omit'});
+
+/**
+ * Fetches the release registry list for SDKs
+ */
+export function useSourcePackageRegistries() {
+  const {isLoading, data} = useApiQuery<ReleaseRegistrySdk>(
+    ['https://release-registry.services.sentry.io/sdks'],
+    {
+      staleTime: Infinity,
+    },
+    API_CLIENT
+  );
```

Comment:
Do we have an OOTB error handling here? We should get a sentry error when this request fails so that we know that versions in the onboarding are defaulting to hardcoded ones (those might be ancient at that point in time).

Reply:
it is a good idea, but if this request fails, sentry should capture it OOTB or? I am not aware of places where we handle `isError`

Reply:
> sentry should capture it OOTB or?

yeah, that's what I'm wondering as well because I don't know by heart if we capture it by default these days

it's pretty important that we log this, otherwise, we won't notice this error is happening and people will just install old versions of the SDK - is there some way we can validate whether we log it or not? just for peace of mind